### ISSUE TITLE
Brendan: As an application architect I would like an 508 compliant AngularJS test page containing all AngularJS/HTML controls used in the project

### ISSUE THREAD

**Issue by mmurthydol** [18F/dol-whd-14c#211]
**Problem Scope**
Existing AngularJS pages need to be made section 508 compliant. ARIA attributes are missing. We should be able to tab to controls, etc. as outlined in acceptance criteria. The 508 code patterns used for the controls in the test page created will be used subsequently to modify existing AngularJS pages. Please note future pages will be written in Angular 4. Please be sure to note any 508 differences across AngularJS and Angular 4 and code/document accordingly. 

**Tasks**
i) Make a list of all the controls used in the project (Angular, non-Angular). Have a quick WHD email (/meeting if email not sufficient) to eyeball this list for omissions. 
ii) Create a sample/test AngularJS "Hello World AngularJS 508 Test" page that contains all the UI control types from (i) 
iii) Make the page from (ii) 508 compliant (please see acceptance criteria below). WHD can provide help with "Trusted tester" spreadsheet related testing prior to your debugging/testing during development (meeting best way to provide this info/assistance).

**ACCEPTANCE CRITERIA:**

i) Demonstrate the above newly created sample/test page is  508 compliant 
   a) Visually using tabs, etc. 
   b) Passes Trusted tester process spreadsheet test plan attached below
   c) Pa11y test scripts cover the test cases from the test plan spreadsheet above and pass successfully.
   d) Document which test cases from test plan spreadsheet are not implementable in Pa11y.
ii) AngularJS 508 compliant components must be able to work in Angular 4 pages in future (after #226 is implemented). 
iii) Document steps needed, esp. notes regarding AngularJS, Angular 4 508 differences.

**Comment by binwang89** [comment]
@mmurthydol for 508 testing,  we are going to use the DHS Trusted Tester process and i will provide the document here. it will include all the check point that we need to pass to make sure each web page is section 508 compliance. 
[Test_Results_Report-blank.xlsx](https://github.com/18F/dol-whd-14c/files/1253240/Test_Results_Report-blank.xlsx)
